cql3/functions/functions.hh

```diff
@@ -29,20 +29,23 @@ namespace functions {
 
 class functions {
     using declared_t = cql3::functions::declared_t;
-    static thread_local declared_t _declared;
-    static std::unordered_multimap<function_name, shared_ptr<function>> init() noexcept;
```

Comment:
It was a peculiar design choice to begin with, to have a class with all its members static, because that is just a namespace with functions/variables, and IMO should be implemented as such.

Reply:
it shouldn't be a namespace because it shouldn't be global in the first place, I think this code was translated from java and may be the reason why

Reply:
yes